<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bugetare</class>
 <widget class="QDialog" name="Bugetare">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="modificare">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>761</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Datele despre buget se pot modifica direct in tabela de mai sus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbuget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adauga buget lunar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addproiect">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adauga proiect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="iesire">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>530</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Iesire</string>
   </property>
  </widget>
  <widget class="QLabel" name="l">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Introduceti date necesare calcularii bugetului.</string>
   </property>
  </widget>
  <widget class="QLabel" name="financiar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Financiar:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cifraafaceri">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cifra de afaceri</string>
   </property>
  </widget>
  <widget class="QLabel" name="costcumparare">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costul de cumparare</string>
   </property>
  </widget>
  <widget class="QLabel" name="resurseumane">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resurse umane:</string>
   </property>
  </widget>
  <widget class="QLabel" name="salariiprime">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costuri salarii si prime</string>
   </property>
  </widget>
  <widget class="QLabel" name="formari">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costuri formari personal</string>
   </property>
  </widget>
  <widget class="QLabel" name="alte1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Alte costuri</string>
   </property>
  </widget>
  <widget class="QLabel" name="tehnologia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tehnologia informatiei:</string>
   </property>
  </widget>
  <widget class="QLabel" name="achizitii">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costuri achizitii</string>
   </property>
  </widget>
  <widget class="QLabel" name="telefonie">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costuri telefonice</string>
   </property>
  </widget>
  <widget class="QLabel" name="alte2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Alte costuri</string>
   </property>
  </widget>
  <widget class="QLabel" name="infrastructura">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Infrastructura:</string>
   </property>
  </widget>
  <widget class="QLabel" name="masini">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costuri cu masinile de companie</string>
   </property>
  </widget>
  <widget class="QLabel" name="chirie">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Costuri cu chiriile</string>
   </property>
  </widget>
  <widget class="QLabel" name="alte3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Alte costuri</string>
   </property>
  </widget>
  <widget class="QLabel" name="investitii">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Investitii</string>
   </property>
  </widget>
  <widget class="QLabel" name="cuantum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cuantumul investitiilor</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cifraafaceriEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="costcumparareEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="salariiprimeEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="alte1Edit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="formariEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="telefonieEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="achizitiiEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="alte2Edit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="chirieEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="masiniEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="alte3Edit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cuantumEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>480</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="salveaza">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salveaza</string>
   </property>
  </widget>
  <widget class="QLabel" name="luna">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Luna corespunzatoare:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="luniBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Ianuarie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Februarie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Martie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Aprilie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mai</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Iunie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Iulie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>August</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Septembrie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Octombrie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Noiembrie</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Decembrie</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="an">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anul corespunzator:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="anEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="anuleaza">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Anuleaza</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
